terraform/aws/analytical-platform-data-production/airflow/variables.tf

```diff
@@ -89,9 +104,19 @@ variable "prod_eks_role_arn" {
   description = "ARN of role used by EKS cluster for Airflow-Prod"
 }
 
-variable "prod_cluster_sg_name" {
+variable "prod_eks_cluster_name" {
+  type        = string
+  description = "Name of cluster for Airflow-Prod"
+}
+
+variable "prod_cluster_additional_sg_id" {
+  type        = string
+  description = "Name of cluster additional security group for Airflow-Prod"
+}
+
+variable "prod_cluster_additional_sg_name" {
```

Comment:
Assuming it's not a tag, I'd use the exported id to retrieve the name via a datasource call:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/security_group#name

Remember, in most data calls, all or most inputs are exported as attributes including name.

Reply:
Same as the other one `prod_cluster_additional_sg_name` is manually defined, the EKS cluster SG is not defined in IaC.